packages/react-ds/src/components/Breadcrumb/Breadcrumb.tsx

```diff
@@ -0,0 +1,30 @@
+import * as React from 'react'
+
+import { IBreadcrumb } from '@juntossomosmais/venice-types'
+import classNames from 'classnames/bind'
+
+import styles from '@venice/styles/components/Breadcrumb.module.scss'
+
+export interface IBreadcrumbProps extends IBreadcrumb {
+  /** string */
+  className?: string
+  /** React Element */
+  children?: React.ReactNode
+}
+
+export const Breadcrumb: React.FunctionComponent<IBreadcrumbProps> = ({
+  children,
+  color = 'default',
```

Comment:
You can use a ENUM to keep VALID_COLORS,
Because if someone pass an invalid color the color `default` can be applied.
Without enum you can set a empty color `styles['non_exciste']` => `null` or call another attr `styles['breadcrumbItem']` it's possible to

Reply:
Communication made by DM